doc/arbtt.xml

```diff
@@ -318,6 +318,20 @@ $ runhaskell Setup.hs install</screen>
       parentheses.
     </para>
 
+    <para>
+      You can assign conditions to variables using
+      <literal>let</literal>:
+      <programlisting>
+        let arbtt = current window $title =~ m/arbtt/ in {
+          $arbtt &amp;&amp; $time &lt; 14:00 ==&gt; tag arbtt-morning,
+          $arbtt &amp;&amp; $time &gt; 14:00 ==&gt; tag arbtt-afternoon
+        }
+      </programlisting>
+      Everything that is a valid condition is assignable and you can
+      reference bound variables in rules by prefixing them with a
+      dollar (<literal>$</literal>).
```

Comment:
Do you have thoughts on whether referring to them with or without `$` is better?

Reply:
I personally like the `$`, because it fits nicely together with other values that are replaced during evaluation like `$title`.

It also makes it easier to distinguish between builtin language keywords like `current window` and a condition binding.

But we can also change it, I have no overly strong opinion, just a slight preference :)